### PR TITLE
feat(blueprint-test): add JSONPathEqs()

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

File: infra/blueprint-test/pkg/golden/golden.go [hunk 1]
```diff
@@ -156,3 +157,16 @@ func (g *GoldenFile) JSONEq(a *assert.Assertions, got gjson.Result, jsonPath str
 	gfData := gf.Get(jsonPath).String()
 	a.Equal(gfData, gotData, fmt.Sprintf("expected %s to match fixture %s", jsonPath, gfData))
 }
+
+// JSONEqs asserts that json content in jsonPaths for got and goldenfile are the same
+func (g *GoldenFile) JSONEqs(a *assert.Assertions, got gjson.Result, jsonPaths []string) {
```
Comment (bharathkkb) [inline infra/blueprint-test/pkg/golden/golden.go:162]: nit: Maybe JSONPathEqs might be a better name?

File: infra/blueprint-test/pkg/golden/golden_test.go [hunk 2]
```diff
@@ -133,3 +133,30 @@ func TestJSONEq(t *testing.T) {
...
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			assert := assert.New(t)
+			os.Setenv(gfUpdateEnvVar, "true")
+			defer os.Unsetenv(gfUpdateEnvVar)
+			got := NewOrUpdate(t, tt.data, tt.opts...)
+			defer os.Remove(got.GetName())
+			got.JSONEqs(assert, utils.ParseJSONResult(t, tt.want), tt.eqPaths)
```
Comment (bharathkkb) [inline infra/blueprint-test/pkg/golden/golden_test.go:159]: Not sure I fully grokked this- is want the newer data version we are testing against? Maybe we should rename the field name?

Also should we add a test case where it does not match with golden?

**Review by bharathkkb — APPROVED** [review]

File: infra/blueprint-test/pkg/golden/golden.go (hunk 1 above)
Comment (apeabody) [reply infra/blueprint-test/pkg/golden/golden.go:162]: SG

File: infra/blueprint-test/pkg/golden/golden_test.go (hunk 2 above)
Comment (apeabody) [reply infra/blueprint-test/pkg/golden/golden_test.go]: I revised the test logic, let me know if it makes more sense now.

Working on how-to include a negative test case as the `JSONPathEqs()` includes an `assert`.

**Pull request merged**